lib/testmap.py

```diff
@@ -143,13 +142,10 @@
             'rhel-8-5',
             'rhel-9-0'
         ],
-        'rhel-8': [
-            'rhel-8-4',
-            'rhel-8-4/firefox',
-        ],
```

Comment:
Did you check that with @henrywang and @teg ? Shouldn't that be moved to 8.5?

Reply:
I checked with @jkozol : 

> I would like to create a 8.5 and a 9 branch instead of using rhel-8

Reply:
Yep. There is now a specific branch for 8.5: https://github.com/osbuild/cockpit-composer/tree/rhel-8-5. The `rhel-8` branch will remain at 8.4. But, then I think instead of removing this test shouldn't it be changed to `rhel-8-5`?

Reply:
>  But, then I think instead of removing this test shouldn't it be changed to rhel-8-5?

Happy to add them of course. I'll first try to trigger them to check they work and then add them to testmap.